sdk/resourcemanager/azure-resourcemanager-cosmos/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 2.28.0-beta.1 (Unreleased)
```

Comment:
You may need a date, as it to be released on branch.

Reply:
Fixed in the new version